static/js/app.js

```diff
@@ -226,7 +234,7 @@ $(function() {
         },
         fetchMarkers : function() {
             if (!this.isReady) return;
-            this.updateUrl();
```

Comment:
It's probably fine, as the URL should keep pointing to the marker ID as long as the info window is open.